requirements_dev.txt

```diff
@@ -6,97 +6,93 @@
 #
 attrs==24.2.0
     # via flake8-bugbear
-bandit==1.7.9
+bandit==1.8.0
     # via -r requirements_dev.in
-black==24.8.0
+black==24.10.0
     # via -r requirements_dev.in
-blinker==1.8.2
+blinker==1.9.0
     # via flask
 boolean-py==4.0
     # via license-expression
 brotli==1.1.0
     # via flask-compress
-build==1.2.1
+build==1.2.2.post1
     # via pip-tools
-cachecontrol[filecache]==0.14.0
+cachecontrol[filecache]==0.14.1
     # via
     #   cachecontrol
     #   pip-audit
 certifi==2024.8.30
     # via requests
-cffi==1.17.0
+cffi==1.17.1
     # via cryptography
-charset-normalizer==3.3.2
+charset-normalizer==3.4.0
     # via requests
 click==8.1.7
     # via
-    #   -r requirements.in
+    #   -r /Users/patrick.sweeting/Projects/moj-flask-govuk-skeleton/requirements.in
     #   black
     #   flask
     #   pip-tools
     #   pur
-coverage[toml]==7.6.1
+coverage[toml]==7.6.9
     # via pytest-cov
-cryptography==43.0.1
+cryptography==44.0.0
     # via pyopenssl
 cssmin==0.2.0
-    # via -r requirements.in
-cyclonedx-python-lib==7.6.0
+    # via -r /Users/patrick.sweeting/Projects/moj-flask-govuk-skeleton/requirements.in
```

Comment:
question - should this have a record, that it was updated via a version on your machine?

Reply:
Yeah looks like your pip compile has had an issue

Reply:
A good question. Github is hiding it from you in the PR view format. If you click to view the full file, you'll see it at the top. Because it didn't get changed Github is not reflecting it in the review layout as it wasn't changed.

Reply:
Updating this thread - on a separate conversation, Pat will be looking at how he ran `pip compile`

Reply:
Figured out your problem @psweeting1. I remember  running into this a while back. There is an open issue on the pip-tools repo: https://github.com/jazzband/pip-tools/issues/2131

The way I resolved it was to remove the annotations using `"--no-annotate"` on the pre-commit-config.